qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -290,7 +304,32 @@ def _set_waveform_params(self, **params_dict: Dict) ->None:
         string += ",".join(
             ["{:7e}".format(params_dict[param]) for param in param_names])
         self.parent.write_raw(string)
+        
+    def _get_duty_cycle(self):
+        """
+        Simple function to extract the duty cycle after checking waveform
```

Comment:
This docstring has ~70% of useful information - "extract the duty cycle after checking waveform"  - and ~30% of useless information - "Simple function to". Could you rephrase it to smth like "Read duty cycle value. Also reads the waveform parameter of the instrument."?

Moreover, one might argue that the fact that this method raises when used for waveforms that don't have a meaningful duty cycle, should also be mentioned in the docstring. In this particular case, this note belongs to the docstring of the parameter (see the `docstring` kwargs for `self.add_parameter('duty_cycle', ...)`.

Finally, this method is fairly short, hence it's code perfectly expresses what is going on, hence the docstring is not really needed, in my humble opinion.

To conclude, please:
* depending on your taste, either remove the docstring or make it "100%" useful :)
* add a notion about "allowed waveforms" to the docstring of the parameter in `add_parameter(...)` call.